public/app.js

```diff
@@ -12,6 +12,9 @@ const configuration = {
   iceCandidatePoolSize: 10,
 };
 
+const callerCandidatesString = 'callerCandidates';
+const calleeCandidatesString = 'calleeCandidates';
```

Comment:
It would be more clear to remove "String" from the end of the variable declaration and make CALLER_CANDIDATES all caps to indicate it's a constant.